pom.xml

```diff
@@ -33,6 +33,9 @@
     <github-branch-source.version>2.8.3</github-branch-source.version>
     <github-plugin.version>1.31.0</github-plugin.version>
 
+    <!-- Library Dependencies Versions -->
+    <commons-io.version>2.7</commons-io.version>
```

Comment:
jenkins core uses https://github.com/jenkinsci/jenkins/blob/2258fb74bb9952728c634ad899bec6a273e959a6/bom/pom.xml#L69

I'm surprised you would need to include it here as it is included in the bom

Reply:
But in the bom 2.6 is included. This causes a conflict in the enforcer. (See also https://issues.jenkins-ci.org/browse/JENKINS-62963). If Jenkins does not update its dependencies more frequently it will cause problems with any library that does update the same dependency more frequently.